docs/source/development.md

```diff
@@ -188,3 +181,35 @@ inv container.push
 inv container.build --push
 ```
 
+## Publishing a release
+
+To publish a release in Github, make sure you are in the main branch, and have
+just tagged the code (see previous section).
+
+Then, you can create a release on [Github](https://github.com/faasm/faabric/releases)
+and publish it from the command line. If it is the first time you are creating
+a release you will have to configure a Github access token (see below).
+
+First, generate a draft release:
+
+```bash
+inv git.release_create
```

Comment:
I don't think this will work, I have a feeling Invoke will replace all underscores with hyphens.